dbms/src/Parsers/ASTInsertQuery.h

```diff
@@ -17,6 +17,10 @@ class ASTInsertQuery : public IAST
     ASTPtr columns;
     String format;
     ASTPtr select;
+
+    // Set to true if the data should only be inserted into attached views
+    bool no_destination = false;
```

Comment:
There is also `Null` table engine.

Reply:
In this case `Kafka` engine table is inserting into itself in a background thread (for materializing views), but I can probably make an internal table and copy attachments from `Kafka` table to it. Would that be better?

Reply:
Interesting... Hold on please, I just started to look at this code.